Amethyst/Managers/WindowManager.swift

```diff
@@ -77,6 +157,53 @@ private struct ObserveApplicationNotifications {
                 windowManager.perform(#selector(WindowManager.applicationActivated(_:)), with: nil, afterDelay: 0.2)
             }
 
+            application.observeNotification(kAXWindowMovedNotification as CFString!, with: application) { accessibilityElement in
+                guard windowManager.userConfiguration.mouseSwapsWindows() else {
+                    return
+                }
+
+                guard let movedWindow = accessibilityElement as? SIWindow else {
+                    return
+                }
+
+                // Do a more in-depth check; make sure the window being dragged is being managed
+                guard let screenManager = windowManager.focusedScreenManager(),
```

Comment:
Update: just tried this code:

```swift
                guard windowManager.windowIsActive(movedWindow) else {
                    print("BUT WINDOW IS NOT ACTIVE")
                    return
                }

                // Do a more in-depth check; make sure the window being dragged is being managed
```

The "more in-depth check" fires, the print statement does not.